### DISCUSSION TITLE
Report | Week 04

### DISCUSSION THREAD

**Discussion by LeonmAndreesen** [SE-TINF22B6/time2shine#24] [Show and tell]
Hi everybody welcome to our weekly report.

This week we started on refining our used tech -stack.
In the meantime we had some progress on using the technology Figma, which helps us by creating fonts for the website. The first screenshot will be published in the next week. 

Additionally we worked on getting done the exercises. We have now getting done especially the sequence diagrams. 

Last but not least, we currently testing first contacts with our API concerning the game.

**Comment by maxschwinghammer** [comment]
Dear Team time2shine,
It would be great if you could upload your tech-stack as well as the sequence diagrams to the post.
